Drop-In/src/main/java/com/braintreepayments/api/NoticeOfCollectionHelper.java

```diff
@@ -0,0 +1,25 @@
+package com.braintreepayments.api;
+
+import android.graphics.Color;
+import android.text.Html;
+import android.text.method.LinkMovementMethod;
+import android.widget.TextView;
+
+public class NoticeOfCollectionHelper {
+
+    private NoticeOfCollectionHelper() {}
+
+    static void setNoticeOfCollectionText(TextView textView, String notificeOfCollectionText) {
+        String privacyPolicyUrlString = "https://www.paypal.com/us/legalhub/home?locale.x=en_US";
+        String html = String.format("<a href=\"%s\">%s</a>", privacyPolicyUrlString, notificeOfCollectionText);
+
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
+            textView.setText(Html.fromHtml(html, Html.FROM_HTML_MODE_COMPACT));
+        } else {
+            textView.setText(Html.fromHtml(html));
+        }
+
+        textView.setMovementMethod(LinkMovementMethod.getInstance());
+        textView.setLinkTextColor(Color.parseColor("#2489F6"));
```

Comment:
This is the same blue we use for our `tintColor` used for this element on iOS incase anyone is wondering where this random hex came from.